.github/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
To encourage having the pull request description before the checklist.

Reply:
We could add it; it works for me. However, we will not always have a description. It will look ugly if we have a "Description" header and ppl keep it empty.

Reply:
We could make it a placeholder for people to replace with the actual description

Reply:
Don in 9eb1b6bb3509ea07184a3274b515e69dc598bc81